sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -50,57 +51,110 @@
         this.service = service;
     }
 
-    Mono<Response<DocumentResultCollection<AnalyzeSentimentResult>>> analyzeSentimentBatchWithResponse(
+    /**
+     * Helper function that analyzes a batch of documents and returns {@link TextAnalyticsPagedFlux} that is a paged
+     * flux contains {@link AnalyzeSentimentResult}.
+     *
+     * @param textInputs A batch of input texts.
+     * @param options The request options, such as the training model version and to show statistics.
+     *
+     * @return {@link TextAnalyticsPagedFlux} of {@link AnalyzeSentimentResult}.
+     */
+    TextAnalyticsPagedFlux<AnalyzeSentimentResult> analyzeSentimentBatch(Iterable<TextDocumentInput> textInputs,
```

Comment:
Can combine `analyzeSentimentBatch` and `anlayzeSentimentBatchWithContext` to be a single method.

Reply:
`analyzeSentimentBatchWithContext` takes context and but `analyzeSentimentBatch` doesn't. Not found a way to make them be a single method. Do you have a suggestion on how?

Reply:
Can't we call `analyzeSentimentBatchWithContext` from `analyzeSentimentBatch` and pass Context as `Context.NONE`, rather than repeating the same code block?

Reply:
See line 67. 
```
return new TextAnalyticsPagedFlux<>(() -> (continuationToken, pageSize) -> withContext(context ->
```

Does passing Context.None helps here?